apps/ai-content-generator/src/components/dialog/common-generator/generatorReducer.tsx

```diff
@@ -0,0 +1,71 @@
+export enum GeneratorAction {
```

Comment:
For me these action names are a bit confusing. The first few map to an event (with verb) but the latter are just nouns. Is there a reason why they are not named as a typical verb actions? We might consider renaming for clarity (makes it easy to see what event actually triggers the action and associated state change)

Reply:
Good catch! I'm going to add verbs to these as well as the actions for the Config page